### ISSUE TITLE
Psyllid should send end-of-run alert

### ISSUE THREAD

**Issue by wcpettus** [project8/psyllid#49]
Related to https://github.com/project8/dragonfly/issues/135, psyllid should send an alert message when it finishes a timed run, that way other control software can do run cleanup.

**Comment by nsoblath** [comment]
@wcpettus is there a particular deadline for this as far as the SC are concerned?

**Comment by wcpettus** [comment]
No.  The current implementation is completely functional, I think this is a generic enhancement that would be nice.

**Comment by laroque** [comment]
We've had several discussions about moving data flow to a push model. Hornet (or a dragonfly replacement) should catch alerts from psyllid to start processing files (solves the issue where file-close triggers are not cross platform). That system would then push to DIRAC (and any local analysis, etc.)
